app/tests/algorithms_tests/test_views.py

```diff
@@ -1367,3 +1369,20 @@ def test_algorithm_model_version_management(settings, client):
     assert not m2.is_desired_version
     del alg.active_model
     assert not alg.active_model
+
+@pytest.mark.django_db
+def test_job_list_row_template_renders(
+    client, settings, django_capture_on_commit_callbacks
+):
+    job = AlgorithmJobFactory()
+
+    assert job.algorithm_image
+
+    job.status = Job.SUCCESS
+    
+    response = render_to_string(
+        template_name="algorithms/job_list_row.html",
+        context={"object": job},
+    )
+
+    assert response.find("View algorithm result") != -1
```

Comment:
I think you would wanna check for the actual link here as well not just the column header.

Reply:
The string is not the column header, but the button title which will not be there if the link is not generated

Reply:
Oh sorry! But I think it still makes sense to check that the button has the expected link set. You could also go one step further and test that all other columns are filled as expected to have a full test. You call the test `test_job_list_row_template_renders`, so I would expect it to test the full template, not just one of the buttons.